### PR TITLE
Fixed integrations secret namespace

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### All Submissions:

Fixed integrations secret namespace

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [x] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes.md if your changes should be included in the release notes for the next release.

### CODE REVIEW

File: pkg/api/v1/project/integration.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func (i Integration) ToAtlas(c client.Client, defaultNS string) (result *mongodb
 			return
 		}
 
-		*target, err = passwordField.ReadPassword(c, defaultNS)
```
Comment (helderjs) [inline pkg/api/v1/project/integration.go:85]: I think you need to change the `ReadPassword()` behavior. It was supposed to fallback to `defaultNS` when the namespace of the requested resource it's not available, however, it's actually uses the passed namespace

**Review by helderjs — CHANGES_REQUESTED** [review]

File: pkg/api/v1/project/integration.go (hunk 1 above)
Comment (igor-karpukhin) [reply pkg/api/v1/project/integration.go:85]: Good point, thanks! Fixed

Comment (helderjs) [reply pkg/api/v1/project/integration.go:85]: There's still something strange. Look at `GetObject()`, it is supposed to already return the correct object key(falling back to parent namespace) but seems to not work correctly.

**Review by helderjs — CHANGES_REQUESTED** [review]

File: pkg/api/v1/project/integration.go (hunk 1 above)
Comment (igor-karpukhin) [reply pkg/api/v1/project/integration.go:85]: Yeah, looks it works the other way around. Let me fix this

**Pull request closed without merging**